schemas/context/enduserids.schema.json

```diff
@@ -9,42 +9,30 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "End User IDs",
   "type": "object",
-  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.",
-  "minProperties": 1,
+  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.\n",
   "properties": {
-    "xdm:realm": {
```

Comment:
Why do we remove `realm`? Is it being replaced with something else?

Reply:
Realm is currently under review internally, the team has asked for it to be removed now and when they have completed the architecture they will make an amended proposal.